app/src/main/java/com/example/musicproject_01/data/model/Genre.java

```diff
@@ -0,0 +1,41 @@
+package com.example.musicproject_01.data.model;
+
+public class Genre {
+    private String mTitle;
+    private int mImage;
+    private String mYear;
+
+    public Genre() {
+
+    }
+
+    public Genre(String Title, int Image, String Year) {
```

Comment:
sai hết rồi này
```suggestion
    public Genre(String title, int image, String year) {
```
Em sửa tương tự ở nơi khác